packages/web-components/src/globals/mixins/connect.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Dispatch, Store } from 'redux';
+import Handle from '../internal/handle';
+
+/**
+ * @param store A redux store.
+ * @returns A funciton that takes a base class and returns a mix-in that connects the component to Redux store.
+ */
+const ConnectMixin = <TState, TStateProps = { [name: string]: any }, TDispatchProps = { [name: string]: any }>(
```

Comment:
Not sure if there's an overlap, wondering if we wanted to use some of the redux mixins from polymer? https://github.com/Polymer/pwa-helpers#redux-helpers

Reply:
@jeffchew I took a look at it before I implemented it myself - And found our that there was no `mapStateToProps()` support (and thus a custom implementation).